### DISCUSSION TITLE
Gotenberg Cloud Run Values

### DISCUSSION THREAD

**Discussion by crivera** [gotenberg/gotenberg#599] [Q&A]
Hi

I was able to deploy a version of Gotenberg to GCP but I was wondering what some of the suggested defaults are that should be used. For reference - 99% of my usecases will be converting docx to pdf.

Here are some defaults:

* Concurrency? 80?
* CPU Limit? 1?
* Memory limit? 512MB? I know the more memory I give it, the more could be handled concurrently- but would it maybe be better to allow concurrency only to be 5 or so and keep memory down? How much memory does an average document need?
* Any Container arguments or commands? Is there somewhere a list of arguments I can see?

Thanks so much for the project and any help on this.

Chris

**Comment by gulien** [comment]
Hello @crivera,

Those are great questions, but I do not have a precise answer for each of them, alas. Best option is to find out yourself by testing your typical payload at scale.

I did some load testing for Gotenberg 6, it may also help you: https://github.com/gotenberg/gotenberg/tree/6.4.4/loadtesting.
